pandas/tests/test_categorical.py

```diff
@@ -2934,26 +2933,28 @@ def test_value_counts_with_nan(self):
             s = pd.Series(pd.Categorical(["a", "b", "a"],
```

Comment:
The test is expecting a `nan` in the construction of the categories if you look at what it is testing.